tests/e2e/specs/gutenberg-plugin.test.js

```diff
@@ -1,24 +1,46 @@
 /**
  * WordPress dependencies
  */
-import { test } from '@wordpress/e2e-test-utils-playwright';
+import { test, expect } from '@wordpress/e2e-test-utils-playwright';
 
 test.describe( 'Gutenberg plugin', () => {
+	// Increasing timeout to 5 minutes because potential plugin install could take longer.
+	test.setTimeout( 300_000 );
+
+	test.beforeAll( async ( { requestUtils } ) => {
+		// Install Gutenberg plugin if it's not yet installed.
+		const pluginsMap = await requestUtils.getPluginsMap();
+		if ( ! pluginsMap.gutenberg ) {
+			await requestUtils.rest( {
+				method: 'POST',
+				path: 'wp/v2/plugins?slug=gutenberg',
```

Comment:
I think `slug` should be part of the `body` since this is a `POST` request.

Reply:
I thought so too originally, but the controller expects slug as a query param.

Reply:
That is a bit confusing, especially after reading the PHPUnit tests - https://github.com/WordPress/wordpress-develop/blob/trunk/tests/phpunit/tests/rest-api/rest-plugins-controller.php#L384-L385.

Reply:
Let's double-check this before finalizing things.

Referencing the endpoint announcement post for the future: https://make.wordpress.org/core/2020/07/16/new-and-modified-rest-api-endpoints-in-wordpress-5-5/

Reply:
It looks like `slug` works as a query argument and as body data.